opts/secret.go

```diff
@@ -50,7 +50,7 @@ func (o *SecretOpt) Set(value string) error {
 
 		value := parts[1]
 		switch key {
-		case "source":
+		case "source", "src":
 			options.Source = value
 		case "target":
```

Comment:
I think there was some discussion about that, because we don't (yet?) support a custom path inside the container, only the name, so having those could be confusing

@ehazlett do you recall?